--- src/core/ngx_regex.h	2012-05-17 14:47:04.000000000 +0100
+++ src/core/ngx_regex.h	2013-01-25 13:43:19.824749681 +0000
@@ -56,5 +56,8 @@
 
 ngx_int_t ngx_regex_exec_array(ngx_array_t *a, ngx_str_t *s, ngx_log_t *log);
 
+void ngx_regex_malloc_init(ngx_pool_t *pool);
+void ngx_regex_malloc_done(void);
+
 
 #endif /* _NGX_REGEX_H_INCLUDED_ */
--- src/core/ngx_regex.c	2012-05-17 14:47:04.000000000 +0100
+++ src/core/ngx_regex.c	2013-01-25 13:42:31.547052651 +0000
@@ -77,7 +77,7 @@
 }
 
 
-static ngx_inline void
+void
 ngx_regex_malloc_init(ngx_pool_t *pool)
 {
 #if (NGX_THREADS)
@@ -95,7 +95,7 @@
 }
 
 
-static ngx_inline void
+void
 ngx_regex_malloc_done(void)
 {
 #if (NGX_THREADS)
